cephalopod/templates/server_list.html

```diff
@@ -317,6 +341,11 @@
         $('#show-enabled').trigger('change');
         $('#show-disabled').trigger('change');
 
+        // Prevents 'Additional data' dropdown to close when clicking in the label
+        $('label.dropdown-item').on('click', function() {
+            $(this).parent().parent().addClass('show');
```

Comment:
can we use `.closest()` with a selector? multiple `parent()` calls are usually not very  obvious/maintainable.